lib/beaker/host.rb

```diff
@@ -202,8 +210,13 @@ def ip
     #Examine the host system to determine the architecture
     #@return [Boolean] true if x86_64, false otherwise
     def determine_if_x86_64
-      result = exec(Beaker::Command.new("arch | grep x86_64"), :acceptable_exit_codes => (0...127))
-      result.exit_code == 0
+      if is_cygwin?
```

Comment:
Basically the same change I made in [PR-694](https://github.com/puppetlabs/beaker/pull/694/files). 

Worth noting that this code is very similiar to [host/windows/pkg.rb#L47](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host/windows/pkg.rb#L47) and will need to be updated if we want to continue supporting windows 2003.